CHANGELOG.md

```diff
@@ -40,8 +40,8 @@
 - Instances are no longer created with an IPC namespace by default. An IPC
   namespace can be specified with the `-i|--ipc` flag.
 - `--bind`, `--nv` and `--rocm` options for `build` command can't be set through
-  environment variables `SINGULARITY_BIND`, `SINGULARITY_BINDPATH`, `SINGULARITY_NV`,
-  `SINGULARITY_ROCM` anymore due to side effects reported by users in this
+  environment variables `APPTAINER_BIND`, `APPTAINER_BINDPATH`, `APPTAINER_NV`,
```

Comment:
No, we do not want to rename all the configuration variables.  We have in our announcement that we weren't going to change these.

Reply:
I invite us to defer this into resolution of #7 and make #7 a blocker for v1.0.0

Reply:
I concur with this idea. Thank you Paul!